plugin/adsk/scripts/mayaUsd_selectionUtils.py

```diff
@@ -0,0 +1,18 @@
+import ufe
+
+def expandPathToSelection(path):
+    """
+    Retrieves the path of the selection item that contains the provided path.
+    If there is no selection or no item in the selection is an ancestor of the item,
+    then the original path itself is returned.
+    """
+    selection = ufe.GlobalSelection.get()
+    ufePath = ufe.PathString.path(path)
+    if selection.containsAncestor(ufePath):
```

Comment:
Ah, the if is redundant with the while... I will fix that. The code is getting shorter that is a good sign 😃